README.md

```diff
@@ -0,0 +1,16 @@
+# url-shortener
+LAP4 Code Challenge to create a url-shortener using a python framework, in this case Django. 
+<h2 align="center"> <img src="https://raw.githubusercontent.com/ABSphreak/ABSphreak/master/gifs/Hi.gif" width="25px"> Welcome to our url-shortener! <img src="https://raw.githubusercontent.com/ABSphreak/ABSphreak/master/gifs/Hi.gif" width="25px"></h2>
+
+## Usage
+
+- `pipenv shell`
+- `pipenv install`
+- `cd urlshortener`
+- `python manage.py runserver`
+
+## Future Features
+
+- Use django to return root url instead of hard coding it in
+- Add validation to url strings entered by user
```

Comment:
Nice clean readme.
Usage instructions worked once I'd cloned the repo.
We used [validators](https://validators.readthedocs.io/en/latest/#module-validators.url) for url validation - super easy to implement!